src/operators.jl

```diff
@@ -249,21 +268,30 @@ for op in ops_2_to_1_ffz
     SparseConnectivityTracer.is_crossder_zero_global(::T)      = true
 end
 
+is_firstder_arg2_zero_local(::typeof(mod), x, y) = ifelse(y > 0, y > x, x > y)
+is_firstder_arg2_zero_local(::typeof(rem), x, y) = ifelse(y > 0, y > x, x > y)
```

Comment:
I find it weird that these two lines are the same

Reply:
I'll take a closer look at `rem` on negative numbers.

Reply:
Removed `rem` for now.